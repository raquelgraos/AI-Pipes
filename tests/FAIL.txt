VE      BC      BB      BB      BB      LH      BB      LH      VE      FB
BD      BE      VC      LV      LV      VB      VE      VB      BC      VC
BC      VE      VB      VC      BE      BC      FD      BD      FB      VE
BD      BE      LH      FE      VC      BE      VE      BE      BB      VC
VB      VC      LH      FB      FD      BE      BD      VC      FC      FB
VB      FE      BD      BE      VE      BC      BC      FD      VB      VC
LH      VB      VC      FC      VE      BD      BB      BB      VC      FE
BD      BC      VE      VB      VB      BD      BC      BC      FD      LV
VD      VE      VD      BE      VD      BE      FB      VB      FE      LV
FD      BC      LV      BC      LH      BD      BC      BC      LV      VD
