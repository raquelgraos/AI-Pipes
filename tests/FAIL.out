VB	BB	BB	BB	LH	VE	VB	BB	VE	FB
FC	BD	VC	VD	VE	BD	BE	VD	BC	BE
FB	BD	LH	FE	FC	FC	FC	FB	VB	BE
BD	BE	FD	LH	BB	LH	VE	BD	BE	FC
FC	VD	BB	VE	VD	FE	BD	BE	VD	FE
FD	LH	VC	BD	VE	FD	VC	VD	FE	FB
FD	VE	VB	BC	BC	LH	LH	LH	VE	LV
FB	VD	BC	BB	BB	LH	LH	BB	BE	LV
BD	LH	VE	BD	BE	VB	LH	BE	BD	VC
FC	FD	BC	BC	VC	FC	FD	BC	BC	FE
